src/pages/actionitems/actionitems.ts

```diff
@@ -202,15 +203,17 @@ export class ActionitemsPage {
   public saveXLS() {
         var headers = this.columns.map(c => c.title);
         var values = this.no.map(nq => {
+							console.log(nq);
                 return [
-                        nq.threadName,
-                        nq.subThreadName,
-                        nq.questionText,
-                        // nq.answers[nq.answers.length - 1].answer,
-                        nq.answers[nq.answers.length - 1].what,
-                        nq.answers[nq.answers.length - 1].when,
-                        nq.answers[nq.answers.length - 1].who,
-                        this.calculateRiskScore(nq.answers[nq.answers.length - 1].likelihood, nq.answers[nq.answers.length - 1].consequence)
+									nq.mrLevel,
+                  nq.threadName,
+                  nq.subThreadName,
+                  nq.questionText,
+                  // nq.answers[nq.answers.length - 1].answer,
+                  nq.answers[nq.answers.length - 1].what,
+                  nq.answers[nq.answers.length - 1].when,
+                  nq.answers[nq.answers.length - 1].who,
+                  this.calculateRiskScore(nq.answers[nq.answers.length - 1].likelihood, nq.answers[nq.answers.length - 1].consequence)
```

Comment:
Let's extract these out, so that it's easier to parse?

Ex.
```
let riskLikelihood = nq.answers[nq.answers.length - 1].likelihood
let riskConsequence = nq.answers[nq.answers.length - 1].consequence
this.calculateRiskScore(riskLikelihood, riskConsequence)
```